src/app/http-interceptors/auth-interceptor.ts

```diff
@@ -0,0 +1,31 @@
+import { Injectable } from '@angular/core';
+import {
+  HttpInterceptor, HttpHandler, HttpRequest
+} from '@angular/common/http';
+
+import { TokenService } from 'angular-aap-auth';
```

Comment:
The new version of the library is called ng-ebi-authorization

Reply:
As we discussed we will updated this in https://trello.com/c/ZVQJnYVF/2-upgrade-to-angular-6.  I'll add comment in the board.